pkg/ride/functions_proto.go

```diff
@@ -154,9 +147,8 @@ func reentrantInvoke(env Environment, args ...rideType) (rideType, error) {
 	}
 
 	res, err := invokeFunctionFromDApp(env, recipient, fnName, listArg)
-
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get Result from invokeFunctionFromDApp")
+		return nil, errors.Wrapf(err, "reentrantInvoke")
```

Comment:
The same as above

Reply:
Not in this case, the errors of `invokeFunctionFromDApp` is quite descriptive, so we have only to add where it happens. Previous error message was not correct.